<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>424</width>
    <height>342</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>424</width>
    <height>342</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>RendeBOX Launcher</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="1" column="0">
     <widget class="QGroupBox" name="groupBox">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="toolTip">
       <string>you can change your render settings, else global sene render settings will apply</string>
      </property>
      <property name="title">
       <string>override settings</string>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>start frame</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="start_frame_LE"/>
         </item>
        </layout>
       </item>
       <item row="0" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>end frame</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="end_frame_LE"/>
         </item>
        </layout>
       </item>
       <item row="1" column="0" colspan="2">
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>output path</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="output_path_LE"/>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QFrame" name="frame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="1" column="0" colspan="2">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>priority</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="priority_CB">
           <property name="minimumSize">
            <size>
             <width>344</width>
             <height>0</height>
            </size>
           </property>
           <item>
            <property name="text">
             <string>Low</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Medium</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>High</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item row="0" column="0" colspan="2">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>description</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="description_LE">
           <property name="toolTip">
            <string>add a unique notes to your render to remember easily</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="3" column="0">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="3" column="1">
        <widget class="QPushButton" name="send_PB">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>50</height>
          </size>
         </property>
         <property name="text">
          <string>send to RenderBOX</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QCheckBox" name="custom_CB">
      <property name="text">
       <string>Use Custom Settings</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>www.technicaldirector.in/renderbox</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>424</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>custom_CB</sender>
   <signal>clicked(bool)</signal>
   <receiver>groupBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>220</x>
     <y>38</y>
    </hint>
    <hint type="destinationlabel">
     <x>220</x>
     <y>94</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
